src/WinGet.RestSource/Models/Schemas/PackageManifest.cs

```diff
@@ -106,7 +106,39 @@ public void UpdateVersion(Version version)
             this.AssertVersionsNotNull();
 
             this.Versions.Update(version);
-        }
+        }
+
+        /// <summary>
+        /// Add Version.
+        /// </summary>
+        /// <param name="version">Version to add.</param>
+        public void AddVersion(VersionExtended version)
+        {
+            // Verify Parameters not null
+            ApiDataValidator.NotNull(version);
+
+            if (this.Versions == null)

```

Comment:
nit: You can simplify this to:

this.Versions ??= new VersionsExtended();

[https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator#examples](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator#examples)